tests/integration/targets/eos_ospf_interfaces/tests/common/merged.yaml

```diff
@@ -0,0 +1,67 @@
+---
+- debug:
+    msg: Start eos_ospf_interfaces merged integration tests ansible_connection={{
+      ansible_connection }}
+
+- include_tasks: _remove_config.yaml
+
+- block:
+
+    - name: merge given ospf interfaces configuration
+      become: true
+      register: result
+      arista.eos.eos_ospf_interfaces: &id001
+        config:
+          - name: "Vlan1"
+            address_family:
+              - afi: "ipv4"
+                dead_interval: 29
+                mtu_ignore: true
+                hello_interval: 66
+              - afi: "ipv6"
+                hello_interval: 77
+                cost: 106
+                transmit_delay: 100
+                ip_params:
+                  - afi: "ipv6"
+                    retransmit_interval: 115
+                    dead_interval: 56
+                    passive_interface: true
+                  - afi: "ipv4"
+                    area:
+                      area_id: "0.0.0.5"
+                    priority: 45
+          - name: "Vlan2"
+            address_family:
+              - afi: "ipv6"
+                ip_params:
+                  - afi: "ipv4"
+                    area:
+                      area_id: "0.0.0.6"
+                    hello_interval: 45
+                    retransmit_interval: 100
+        state: merged
+
+    - become: true
+      arista.eos.eos_facts:
+        gather_network_resources: ospf_interfaces
+
+    - assert:
+        that:
+          - result.commands|length == 16
+          - result.changed == true
+          - result.commands|symmetric_difference(merged.commands) == []
+          - result.after|symmetric_difference(ansible_facts['network_resources']['ospf_interfaces']) == []
```

Comment:
Asserting before/after from gathered facts is not much useful, imo. Both are generated from the same code path.

Reply:
I might have to change this assertion for most of the interfaces related tests. I shall take it up as a separate PR.